File: hier/project-euler/euler-001/euler1.cs
// euler1.cs - Project Euler problem 1: Multiples of 3 and 5

using System;
using System.Linq;

[assembly:System.Reflection.AssemblyVersion("0.1.0.0")]

internal static class Program {
    private static void Main()
    {
        var n = 0;

        foreach (var k in Enumerable.Range(0, 1000))
            if (k % 3 == 0 || k % 5 == 0) n += k;

        Console.WriteLine(n);
    }
}

File: hier/project-euler/euler-001/euler1.cxx
// euler1.cxx - Project Euler exercise - Multiples of 3 and 5

#include <iostream>

int main()
{
    int n {0};
    for (int i {1}; i != 1000; ++i)
        if (i % 3 == 0 || i % 5 == 0) n += i;

    std::cout << n << '\n';
}